src/dependency-details-retriever.ts

```diff
@@ -8,7 +8,7 @@ import { FetchHttpClient, HttpClient } from './http';
 
 const REQUEST_DELAY_MS = 1400;
 const RELEVANT_LABELS = ['good+first+issue', 'help+wanted', 'documentation'];
-const MIN_ISSUE_DATE = moment().subtract(365, 'days').format('YYYY-MM-DD');
+const MIN_ISSUE_DATE = DateTime.local().minus({ days: 365 }).toFormat('yyyy-LL-dd');
```

Comment:
I wondered about that. You're correct, I was looking at what was done elsewhere because I'm more familiar with Moment than Luxon. I'll take a look at making that change and give the timezones some more thought. I didn't examine the context of this variable very closely.

Reply:
Actually, besides the issue of being clearer, it looks like [since](https://docs.github.com/en/rest/reference/issues#list-issues-assigned-to-the-authenticated-user--parameters) is supposed to be ISO 8601, which is needs a timezone anyway.
If we use that as the string, it shouldn't really matter which timezone the object is created with, since (if it works like Moment), it should handle the timezone conversion.

I think if we create this DateTime as utc, the other instantiations should be utc also to be consistent (so for the moment, I left as local). I can change the rest if you'd prefer.

Reply:
This looks great. And I agree with you both that it doesn't matter whether we use .utc() or .local() in this case. And that we should be consistent. I thought we were, but as you point out, we're not. So how you have it looks good.